README.md

```diff
@@ -11,8 +11,21 @@
 [![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=ruuvi_ruuvi.firmware.c&metric=sqale_index)](https://sonarcloud.io/dashboard?id=ruuvi_ruuvi.firmware.c)
 
 Ruuvi Firmware version 3. Built on top of Nordic SDK 15, uses both Ruuvi and external repositories as submodules.
-Under development, please follow [Ruuvi Blog](https://blog.ruuvi.com) for details. The project is in alpha stage,
-build maybe broken in obvious ways and not build at all or in non-obvious ways and fail on some Ruuvi boards. 
+Under development, please follow [Ruuvi Blog](https://blog.ruuvi.com) for details. 
+```
```

Comment:
The advertising startup intervals are probably a lot faster, or at least they should be. I also wouldn't fix this information in readme, but rather autogenerate it to release from the headers.